examples/complete/main.tf

```diff
@@ -135,7 +133,7 @@ module "eks_cluster" {
   context = module.this.context
 }
 
-module "eks_node_group" {
+module "eks_nix_node_group" {
```

Comment:
Unsure why this is modified

Reply:
I added an eks_nt_node_group so it was separating the groups; I was going to ask what the opinion was. could be a separate example instead. As the cluster change is necessary and I was reverting this to be 99% back to its original test sample I figured I'd lower the threshold on this pass.

Reply:
I'm not sure i understand why the module name has changed tho. Could we change it back?

Reply:
Because the windows module was eks_nt_node_group. 
If its that much of an issue I'll revert it when I get out of this bike shed 😉

Reply:
All done